### PR TITLE
Added release notes for added tvOS support for Analytics and Media Analytics

### PR DESCRIPTION
**Pull request by addb**
Alsodded column in current version AEP SDK table to show support for tvOS per extension

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

**Review by stcheng-adobe — APPROVED** [review]

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 1]
```diff
@@ -66,29 +66,29 @@ The Swift iOS SDKs are open source - read more about [our move to Swift and open
...
+| [Adobe Journey Optimizer - Decisioning](../../using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md) |  |  | [![Cocoapods](https://img.shields.io/cocoapods/v/AEPOptimize.svg?color=orange&label=AEPOptimize&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/AEPOptimize) | [Link](https://github.com/adobe/aepsdk-optimize-ios) |
+| [Adobe Target](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target) |  |  | [![Cocoapods](https://img.shields.io/cocoapods/v/AEPTarget.svg?color=orange&label=AEPTarget&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/AEPTarget) | [Link](https://github.com/adobe/aepsdk-target-ios) |
+| [Adobe Campaign Standard](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaign-standard) |  |  | [![Cocoapods](https://img.shields.io/cocoapods/v/AEPCampaign.svg?color=orange&label=AEPCampaign&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/AEPMCampaign) | [Link](https://github.com/adobe/aepsdk-campaign-ios) |
+| [Adobe Campaign Classic](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic) |  |  | To Be Determined |  |
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:91]: Can we update the ACP table to the same format for tvOS support?

Comment (addb) [reply resources/upgrading-to-aep/current-sdk-versions.md:91]: Updated!

File: using-mobile-extensions/adobe-analytics/analytics-faq.md [hunk 2]
```diff
@@ -59,3 +68,24 @@ In the mobile property (tag) in the Data Collection UI, select the property from
 #### Contact Adobe Customer Care <a id="contact-adobe-customer-care"></a>
 
 If you are unable to resolve your concerns through resources provided here, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support) for immediate assistance.
+
+----
+
+### __ is unavailable in application extension for tvOS
+
+This is an issue with the Xcode. When using the AEP SDK from a tvOS app target, there is a compilation error which says "__ is unavailable in application extension for tvOS" even when not building for a tvOS app extension target/app, which is not an intended behavior. We have filed a bug with Apple to get this issue resolved. In the meantime, the workaround for this issue is to mark the classes or functions with the attribute: `@available(tvOSApplicationExtension, unavailable)` to suppress the error.
```
Comment (cdhoffmann) [inline using-mobile-extensions/adobe-analytics/analytics-faq.md:76]: When using the Adobe Experience Platform SDK from a tvOS app target, there is a compilation error which says: "'x' is unavailable in application extension for tvOS", even when you are not building for a tvOS app extension target. This is not intended behavior, and we have filed a bug with Apple to get this issue resolved. In the meantime, ...

**Review by cdhoffmann — APPROVED** [review]
Just a few comments.

File: resources/frequently-asked-questions/README.md [hunk 3]
```diff
@@ -142,3 +142,24 @@ Yes, offline tracking is supported by default when sending XDM Experience events
 
 * Visit the SDK [community forum](https://forums.adobe.com/community/experience-cloud/platform/launch/sdk) to ask questions
 * Contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support) for immediate assistance
+
+## AEP Swift SDKs tvOS <a id="tvos-faqs"></a>
+
+###  'X' is unavailable in application extension for tvOS
```
Comment (cdhoffmann) [inline resources/frequently-asked-questions/README.md:148]: I think there is an extra space at the front right?

File: resources/frequently-asked-questions/README.md [hunk 4]
```diff
@@ -142,3 +142,24 @@ Yes, offline tracking is supported by default when sending XDM Experience events
 
 * Visit the SDK [community forum](https://forums.adobe.com/community/experience-cloud/platform/launch/sdk) to ask questions
 * Contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support) for immediate assistance
+
+## AEP Swift SDKs tvOS <a id="tvos-faqs"></a>
```
Comment (cdhoffmann) [inline resources/frequently-asked-questions/README.md:146]: Maybe just Using AEP Swift SDKs with tvOS

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 5]
```diff
@@ -66,57 +66,57 @@ The Swift iOS SDKs are open source - read more about [our move to Swift and open
...
+| [Identity](../../foundation-extensions/mobile-core/identity/) | ✔️ |  | Bundled in Mobile Core | — |
+| [Signal](../../foundation-extensions/mobile-core/signals/) | ✔️ |  | Bundled in Mobile Core | — |
+| [Lifecycle](../../foundation-extensions/mobile-core/lifecycle/) | ✔️ |  | Bundled in Mobile Core | — |
+| [Rules Engine](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/rules-engine) | ✔️ |  | Bundled in Mobile Core | — |
+| [Adobe Experience Platform Edge Network](../../foundation-extensions/experience-platform-extension/) |  |  | Not Available | — |
+| [Identity for Edge Network](../../foundation-extensions/identity-for-edge-network/) |  |  | Not Available | — |
+| [Consent for Edge Network](../../foundation-extensions/consent-for-edge-network/) |  |  | Not Available | — |
+| [Adobe Experience Platform Assurance](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-assurance) | ✔️ |  | [![Cocoapods](https://img.shields.io/badge/AEPAssurance-v1.1.3-orange?logo=apple&style=flat-square)](https://cocoapods.org/pods/AEPAssurance) | [Link](https://github.com/Adobe-Marketing-Cloud/acp-sdks/tree/master/iOS/AEPAssurance) |
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:110]: Assurance should not be marked for tvOS here

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 6]
```diff
@@ -66,57 +66,57 @@ The Swift iOS SDKs are open source - read more about [our move to Swift and open
...
+| [Adobe Audience Manager](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager) |  |  | [![Cocoapods](https://img.shields.io/cocoapods/v/ACPAudience.svg?color=orange&label=ACPAudience&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/ACPAudience) | [Link](https://github.com/Adobe-Marketing-Cloud/acp-sdks/tree/master/iOS/ACPAudience) |
+| [Adobe Analytics - Mobile Services](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services) | ✔️ |  | [![Cocoapods](https://img.shields.io/cocoapods/v/ACPMobileServices.svg?color=orange&label=ACPMobileServices&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/ACPMobileServices) | [Link](https://github.com/Adobe-Marketing-Cloud/acp-sdks/releases/tag/v1.0.0-ACPMobileServices) |
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:116]: Same for mobile services

**Review by emdobrin — APPROVED** [review]

**Pull request merged**